### DISCUSSION TITLE
Preview window on right side of code instead of below it

### DISCUSSION THREAD

**Discussion by stevenarm** [codesandbox/codesandbox-client#4848] [General]
When I open my sandbox in Chrome, the browser preview window in below the code.  When I open it Brave or Firefox, the preview window is on the right side of the code where I prefer it.

I can't find a setting that controls this, clearing site data and cookies for codesandbox didn't help.  Does anyone know how to resolve this?

**Comment by garethx** [comment]
It's set automatically depending on the viewport size, which I guess could be slightly different between browsers. But you can manually switch it from: View > Editor Layout > Flip full layout
